committeeoversightapp/management/commands/import_data.py

```diff
@@ -101,26 +89,26 @@ def add_senate_hearings(self):
                 category = row[9]
 
                 if name:
-                    #save hearing
-                    event = Event(jurisdiction_id = jurisdiction_id,
+                    # get or create hearing
+                    event, created = Event.objects.get_or_create(jurisdiction_id = jurisdiction_id,
                                     name = name,
                                     start_date = start_date,
                                     classification = classification)
-                    event.save()
-
-                    # save committee data (TK)
-                    # for committee in committees:
-
-                    # save event source
-                    source = EventSource(event=event, note="spreadsheet", url=source)
-                    print(source)
-                    source.save()
-
-                    # save category data
-                    try:
-                        category = HearingCategory(event=event, category_id=category)
-                        category.save()
-                    except IntegrityError:
-                        bad_rows.append("Unrecognized category on Senate row " + str(row_count) + ": " + event.name)
+                    if created:
+                        event.save()
+
+                        # save committee data (TK)
+                        # for committee in committees:
+
+                        # save event source
+                        source = EventSource(event=event, note="spreadsheet", url=source)
+                        source.save()
+
+                        # save category data
+                        try:
+                            category = HearingCategory(event=event, category_id=category)
+                            category.save()
+                        except IntegrityError:
+                            bad_rows.append("Unrecognized category on Senate row " + str(row_count) + ": " + event.name)
```

Comment:
Can you tell me a little about this `bad_rows.txt` file? Is it something that we plan to share with Jamie?